File: solutions/swift/resistor-color/1/Sources/ResistorColor/ResistorColor.swift
enum ResistorColor {
    static let colors = [
        "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white",
    ]
    static func colorCode(for: String) throws -> Int { Self.colors.firstIndex(of: `for`) ?? -1 }
}

File: solutions/clojure/resistor-color/1/src/resistor_color.clj
(ns resistor-color)

(def colors
  ["black" "brown" "red" "orange" "yellow"
   "green" "blue" "violet" "grey" "white"])

(def color-codes
  (zipmap colors (range)))

(defn color-code [color]
  (color-codes color))
